src/Dfe.PlanTech.Application/Responses/Queries/GetLatestResponsesQuery.cs

```diff
@@ -54,6 +54,16 @@ private IQueryable<Submission> GetCurrentSubmission(int establishmentId, string
             .Where(IsMatchingIncompleteSubmission(establishmentId, sectionId, completedSubmission))
             .OrderByDescending(submission => submission.DateCreated);
 
+    public async Task ViewLatestSubmission(int establishmentId, string sectionId)
+    {
+        var currentSubmission = GetCurrentSubmission(establishmentId, sectionId, true).FirstOrDefault();
```

Comment:
Any reason we're not using `db.FirstOrDefaultAsync` here?

Reply:
I forgot 😄 
thank you, fixed